### ISSUE TITLE
How to move down daysStackView under Month title

### ISSUE THREAD

**Issue by asadqazi** [Yalantis/YACalendar#2]
How to move down daysStackView under Month title and there should be 2 button to move to next and previous months

**Comment by Vodolazkyi** [comment]
Hi @asadqazi !
To implement this you need:
1. Set ```calendarView.config.month.showTitle = false```
2. Add your custom view with a label for month title and next/previous buttons.

In order to know the current month, you can by implementing CalendarViewDelegate with ```func didUpdateDisplayedDate(_ date: Date)``` method

To scroll next/previous month use ```func scroll(to date: Date)``` method

**Comment by asadqazi** [comment]
and can you please also tell me how to increase size of text and selection color?

**Comment by asadqazi** [comment]
I disabled the month and days view but it leaves a blank space at top.
How to remove that blank space? And also please tell me how to increase the size of selection circles? 

<img width="288" alt="Screenshot 2020-02-28 at 3 53 22 PM" src="https://user-images.githubusercontent.com/11749438/75542909-87265880-5a42-11ea-8d66-df433e6fc937.png">

**Comment by Vodolazkyi** [comment]
@asadqazi update pod to ```0.0.2``` version

**Comment by asadqazi** [comment]
I downloaded the code yesterday.
I am customizing it according to my requirments. But now just stuck at selection size and extra top space

**Comment by asadqazi** [comment]
And `func scroll(to date: Date)` not working when going to previous month on button click

**Comment by Vodolazkyi** [comment]
@asadqazi I have fixed extra top size today. What do you mean by selection size? I will check scroll(to date: Date)

**Comment by Vodolazkyi** [comment]
@asadqazi I just checked and scroll works fine 
```swift
calendarView.currentDate = Calendar.current.date(byAdding: .month, value: -1, to: calendarView.currentDate)!
calendarView.scroll(to: calendarView.currentDate)

```
Maybe you have another case?

**Comment by asadqazi** [comment]
I am talking about this selection size
![Untitled](https://user-images.githubusercontent.com/11749438/75561617-2100fc00-5a69-11ea-9edc-ece8abb5d7f0.png)

**Comment by asadqazi** [comment]
The day selection, range color is static.
I think it can ne configurable. Like for day config

**Comment by asadqazi** [comment]
I tried to inherit the day view class and it gives me error because of a non open class. 
Searched SO and found
[https://stackoverflow.com/a/38979479/3366100](url)

<img width="947" alt="Screenshot 2020-03-01 at 1 48 58 PM" src="https://user-images.githubusercontent.com/11749438/75622561-718c6c80-5bc3-11ea-895b-6967a8250e05.png">

**Comment by asadqazi** [comment]
I locally changed the classes to open but another issue that from stroryboard the CalendarView refrence is nil while running the app

**Comment by asadqazi** [comment]
Please create a project and use your lib as pod, and see if problem occurs on your side.
Because currently you locally added it as framework

**Comment by Vodolazkyi** [comment]
@asadqazi I have fixed the issues. Please check version 0.0.3

**Comment by asadqazi** [comment]
There should also be an option so that user cannot select futures dates from today.
I set the `endDay` to `Date()` but it creates an issue.

**Comment by Vodolazkyi** [comment]
@asadqazi Did you try ```func disableDays(with dates: [Date])``` method?

**Comment by asadqazi** [comment]
Yes but it take an array of dates and I want to disable all the remaining future days from today.
According to this method I will have to get all the future days `Date` objects and will use them.
